docs/CONFIG-VARS.md

```diff
@@ -306,7 +306,7 @@ Cluster-autoscaler is currently only used for AWS EKS clusters. GCP GKE and Azur
 | INGRESS_NGINX_NAMESPACE | ingress-nginx helm install namespace | string | ingress-nginx | false | | baseline |
 | INGRESS_NGINX_CHART_URL | ingress-nginx helm chart url | string | https://kubernetes.github.io/ingress-nginx | false | | baseline |
 | INGRESS_NGINX_CHART_NAME | ingress-nginx helm chart name | string | ingress-nginx | false | | baseline |
-| INGRESS_NGINX_CHART_VERSION | ingress-nginx helm chart version | string | 3.20.1| false | | baseline |
+| INGRESS_NGINX_CHART_VERSION | ingress-nginx helm chart version | string | "UNSPECIFIED" | false | If left as "UNSPECIFIED", version 3.20.1 will be used for K8s clusters whose version is <= 1.21.X and version 4.0.13 will be used for K8s clusters whose version is >= 1.22.X | baseline |
```

Comment:
Currently ingress-nginx v1.1.0 supports kubernetes 1.19-1.22 why would we care at this point? Viya no longer supports kubernetes v1.18 so the code here should not either. Always moving forward. If we have to make a release that break compatibility that's not an issue. Folks can pull older versions of the code base.